tests/acceptance/serverd-multi-versions.sh

```diff
@@ -49,7 +49,9 @@ function install(){
     rm -rf "$1"
     if test -z "$2"
     then
-        make install prefix=$PWD/$1/var/cfengine 2>&1 >/dev/null
```

Comment:
It is normal to have `prefix=` in the `make install` command. Also `destdir=` is even more common. So these should still work.

Reply:
This would only be possible if we started searching for our binaries in ``$PATH`` or something like that. Otherwise specifying ``prefix=`` in ``make install`` changes the paths too late for our code to adapt to it. Or do you have any suggestions how to make this work? Everything that comes to my mind is just worse than not supporting ``prefix=`` in ``make install``.